Sources/Rendering/Core/VolumeMapper/index.d.ts

```diff
@@ -233,6 +238,19 @@ export interface vtkVolumeMapper extends vtkAbstractMapper {
 	update(): void;
 }
 
+/**
+ * create the piecewise and the color transfer functions for the radon blending.
+ * @static
+ * @param {object} values: Store all needed values to create the CTF:
+ * {
+ *   min: Define minimum voxel value
+ *   minValue: Value that will be attribute to voxels with min value
+ *   max: Define maximum voxel value
+ *   maxValue: Value that will be attribute to voxels with max value
+ * }
```

Comment:
Please provide more context and help to choose the parameter values.
(e.g. to be used with the RADON_TRANSFORM blend mode, hounsfield units, CT volume...)
Talk about absorption...
Why in your example minValue is > than maxValue ?

Reply:
This is an error